diff/merge_allof_unit_test.go

```diff
@@ -0,0 +1,1281 @@
+package diff
```

Comment:
Better to put tests in package diff_test

I suggest to rename this file to 'merge_allof_test.go'